### PR TITLE
Add creation date and enhanced networking support for images

### PR DESCRIPTION
**Pull request by puneetloya**
Have added creationDate and enaSupport members to image so as to know the images created date and if they support enhanced networking.

See: 
* http://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_Image.html
* http://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeImages.html
* https://github.com/fog/fog-aws/blob/7e5e6ff/lib/fog/aws/parsers/compute/describe_images.rb

Fixes: https://github.com/fog/fog-aws/issues/320

### CODE REVIEW

**Comment by lanej** [comment]
@puneetloya can you rebase off of current master ?

**Comment by puneetloya** [comment]
@lanej Cool will do that..

**Comment by lanej** [comment]
Fixes https://github.com/fog/fog-aws/issues/320

File: lib/fog/aws/parsers/compute/describe_images.rb [hunk 1]
```diff
@@ -66,9 +66,9 @@ def end_element(name)
               end
             else
               case name
-              when 'architecture', 'description', 'hypervisor', 'imageId', 'imageLocation', 'imageOwnerAlias', 'imageOwnerId', 'imageState', 'imageType', 'kernelId', 'name', 'platform', 'ramdiskId', 'rootDeviceType','rootDeviceName','virtualizationType'
+              when 'architecture', 'description', 'hypervisor', 'imageId', 'imageLocation', 'imageOwnerAlias', 'imageOwnerId', 'imageState', 'imageType', 'kernelId', 'name', 'platform', 'ramdiskId', 'rootDeviceType','rootDeviceName','virtualizationType','creationDate'
```
Comment (lanej) [inline lib/fog/aws/parsers/compute/describe_images.rb]: You may wish to separate `creationDate` and parse to a `Time` object.  I cannot find an example of the `creationDate` format so it's hard to determine if it's feasible.

File: lib/fog/aws/requests/compute/describe_images.rb [hunk 2]
```diff
@@ -36,6 +36,8 @@ class Real
         #       * 'rootDeviceName'<~String> - Root device name, e.g. /dev/sda1
         #       * 'rootDeviceType'<~String> - Root device type, ebs or instance-store
         #       * 'virtualizationType'<~String> - Type of virtualization
+        #       * 'creationDate'<~Date> - Date the image was created
```
Comment (lanej) [inline lib/fog/aws/requests/compute/describe_images.rb]: `<~Date>` isn't actually true.  It is a `<~String>` until the parser comment is addressed.

File: lib/fog/aws/parsers/compute/describe_images.rb [hunk 3]
```diff
@@ -68,7 +68,7 @@ def end_element(name)
               case name
               when 'architecture', 'description', 'hypervisor', 'imageId', 'imageLocation', 'imageOwnerAlias', 'imageOwnerId', 'imageState', 'imageType', 'kernelId', 'name', 'platform', 'ramdiskId', 'rootDeviceType','rootDeviceName','virtualizationType','creationDate'
                 @image[name] = value
-              when 'isPublic' or 'enaSupport'
+              when 'isPublic','enaSupport'
```
Comment (lanej) [inline lib/fog/aws/parsers/compute/describe_images.rb]: I would squash this commit into the commit that added the ` or `

File: lib/fog/aws/parsers/compute/describe_images.rb (hunk 1 above)
Comment (puneetloya) [inline lib/fog/aws/parsers/compute/describe_images.rb]: The SDK returns name this way "2014-12-01T19:31:49.000Z". Can I convert the time format to '%FT%TZ' is that fine for time format, or  is there another standard format I need to use?

Comment (lanej) [inline lib/fog/aws/parsers/compute/describe_images.rb]: [`Time.parse` should do it for you](https://github.com/fog/fog-aws/blob/09a405cc1474636f436406fc31c7b8baebaead8f/lib/fog/aws/parsers/cloud_formation/list_stacks.rb#L21)

Comment (puneetloya) [inline lib/fog/aws/parsers/compute/describe_images.rb]: fixed it!!

File: lib/fog/aws/requests/compute/describe_images.rb (hunk 2 above)
Comment (puneetloya) [inline lib/fog/aws/requests/compute/describe_images.rb]: fixed it!!

**Comment by lanej** [comment]
@puneetloya please add the relevant changes to [the image tests](http://github.com/fog/fog-aws/blob/c4b619d0f4ccb622921aa53579f95c29b7a73851/tests/requests/compute/image_tests.rb#L1) and we should be good to merge.

**Comment by puneetloya** [comment]
ready to merge I guess?

**Comment by lanej** [comment]
Thanks @puneetloya

**Comment by puneetloya** [comment]
when is the next release planned for (new tag version)?

**Comment by lanej** [comment]
https://rubygems.org/gems/fog-aws/versions/0.13.0

**Comment by geemus** [comment]
Thanks!

**Comment by puneetloya** [comment]
Thanks a lot!

**Pull request merged**